### PR TITLE
11.0 port manawi

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

**Review by Railia — APPROVED** [review]

File: reminder_task_deadline/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This module don't need to be configured.
+
+
+Usage
+=====
+
+* Open menu ``{Project} >> {Tasks}``, click on any Task
+* Click ``[{Edit}]``, set the time data in ``[{Deadline}]`` field
+* In ``[{Reminders}]`` choose one of the standarts reminders or create a new one
```
Comment (ilmir-k) [inline reminder_task_deadline/doc/index.rst]: Use correct formatting according new template

**Comment by iledarn** [comment]
@ilmir-k , done

**Comment by ilmir-k** [comment]
@iledarn there are lint errors to be fixed

**Review by yelizariev — APPROVED** [review]

**Comment by iledarn** [comment]
@ilmir-k , done lints

**Review by ilmir-k — APPROVED** [review]

**Pull request merged**